### ISSUE TITLE
Bug: Counter shows known devices to UniFi, not Active devices

### ISSUE THREAD

**Issue by cjramseyer** [clyra/unifics#24]
I recently tried this integration, and it connects to my UniFi controller, and retrieves information.
However, I found that it counts 'known' devices to unifi, which is any device that has ever connected whether active or not.

Like many other trying to use this integration, I need 'Active' devices, not known.

In testing, I connected a mobile device to the 'guest' network (where I am seeking this for), the count increases from 0 to 1 (great!).
I disconnected the device from that network, allow the DHCP lease to expire (24 hours), and UniFi still knows the device, the count remains at 1, until I login to UniFi and tell the controller to 'Forget Device'.

**Comment by clyra** [comment]
Hm... i guess this is a unifi problem, because what you see at the web interface is what the sensor gets. Did you tried to lower the dhcp lease time?

**Comment by cjramseyer** [comment]
Yes, I reduced the lease time to less than an hour and the issue persists.

**Comment by clyra** [comment]
I'm not sure if there's a fix for that... the integration uses the pyunifi python package (https://github.com/finish06/pyunifi). Their documentations says that the function get_clients(self) "Return a list of all active clients, with significant information about each". Perhaps those "signficant information" has the last time the device sent something, and maybe we can use that, at the risk of considering a device that is quiet as non-active.
